decentraland/unity-renderer
# feat: [Explore V2] Remove the old Explore feature

## What does this PR change?
- [x] Fix #1509 

Remove all the code and assets related to the old Explore feature.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/Remove-Old-ExploreHUD-Feature
2. Check everything is working fine.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
